library/spdm_requester_lib/libspdm_req_get_digests.c

```diff
@@ -107,7 +129,7 @@ libspdm_return_t libspdm_try_get_digest(void *context, uint8_t *slot_mask,
 
     libspdm_zero_mem(spdm_response, spdm_response_size);
     status = libspdm_receive_spdm_response(
-        spdm_context, NULL, &spdm_response_size, (void **)&spdm_response);
+        spdm_context, session_id, &spdm_response_size, (void **)&spdm_response);
     if (LIBSPDM_STATUS_IS_ERROR(status)) {
         goto receive_done;
     }
```

Comment:
Comment for line 186, I dont think we need append_message_b if it is in session.